models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -506,4 +506,33 @@ public function getReturnTypeDeclaration(): ?string
     {
         return 'array';
     }
+
+    /**
+     * @return string|null
+     */
+    public function getPhpdocInputType(): ?string
+    {
+        if ($this->getPhpdocType()) {
+            return "\\". $this->getPhpdocType();
```

Comment:
```suggestion
            return $this->getPhpdocType();
```
leading backslash not required here. please see https://github.com/pimcore/pimcore/blob/b1fa740f699e4fde71b7526f06c538f83be8065c/models/DataObject/ClassDefinition/Data/Extension/Relation.php#L33-L40